src/Accordion/accordion.js

```diff
@@ -44,22 +44,18 @@ class Accordion extends Component {
             activeItems: activeItems,
         });
 
-        if (this.props.onChange) {
-            this.props.onChange(this.props.accordion ? activeItems[0] : activeItems);
-        }
+        this.props.onChange(this.props.accordion ? activeItems[0] : activeItems);
     }
 
     renderItems() {
         const { accordion, children } = this.props;
-        if (!children) {
-            return null;
-        }
 
         return React.Children.map(children, (item, index) => {
             const key = index;
             const expanded = (this.state.activeItems.indexOf(key) !== -1) && (!item.props.disabled);
 
             return React.cloneElement(item, {
+                disabled: item.props.disabled,
```

Comment:
I don't see `disabled` as a prop on `AccordionItem` so I'm not sure why it is passed there? This creates weirdness in the snapshots where, since there is no default value, it will be set to `undefined`.